imaging/ml/ml_codelab/README.md

```diff
@@ -1,14 +1,14 @@
-# ML Codelabs on Cloud Healthcare API
+# Imaging ML Codelabs on Cloud Healthcare API
 
-This directory contains ML codelabs on top of the Cloud Healthcare API. They are intended to be run on Cloud Datalab. Follow these instructions to get started.
+This directory contains Imaging ML Codelabs on top of the Cloud Healthcare API. They are intended to be run on Imaging ML Codelabs. Follow these instructions to get started.
```

Comment:
they are intended to be run in an AI Platform Notebook.

Reply:
done